packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,205 @@
+//
+// SIDE-NAV
+//
+
+@use "../mixins/focus-ring" as *;
+
+// WIP, naming, etc. will be refined... TODO: Add JSON for color tokens, etc.
+:root {
+  /* Sizes: */
+  --token-side-nav-item-padding-horizontal: 12px;
+
+  /* Colors (WIP): */
+  --token-side-nav-foreground-primary: #dedfe3;
+  --token-side-nav-foreground-primary-strong: #fff;
+  --token-side-nav-foreground-faint: #8c909c;
+
+  // Non-active "Links"
+  --token-side-nav-surface-default: #0c0c0e;
+  --token-side-nav-surface-hover: #3b3d45;
+  --token-side-nav-surface-active: #656a76;
+
+  // "Active" "Links"
+  --token-side-nav-selected-surface-default: #656a76;
+  --token-side-nav-selected-surface-hover: #3b3d45;
+  --token-side-nav-selected-surface-active: #656a76;
+}
+
+
+// * SideNav Parent (wrapper) component
+// -------------------------------------------------------------------
+
+.hds-side-nav {
+  display: flex;
+  flex-direction: column;
+  width: 280px;
+  height: 100vh;
+  color: var(--token-side-nav-foreground-primary); // Delete?
+  background: var(--token-side-nav-surface-default);
+}
+
+// Child "wrapper" blocks & child components:
+
+// * Sub-component: Header
+// ---------------------------------------
+.hds-side-nav__header {
+  display: flex;
+  gap: 8px; // TODO: replace with margins
+  align-items: center;
+  padding: 12px;
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+
+  .hds-side-nav__header-logo {
+    margin-right: auto; // Q: Make more generic?
+  }
+
+  // TEMP HACK ONLY! (TODO: replace)
+  .hds-disclosure__toggle:not(:hover) * {
+    color: var(--token-side-nav-foreground-primary) !important;
+  }
+}
+
+.hds-side-nav__header-logo {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  min-width: 48px;
+  height: 48px;
+  padding: 4px;
+  color: var(--token-side-nav-foreground-primary);
+  background: var(--token-side-nav-surface-default); // Q: Should this be added here or should it just be transparent?
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  &:focus,
+  &.mock-focus {
+    @include hds-focus-ring-with-pseudo-element();
+  }
+}
+
+// Some styles in cloud-ui, interfere so adding additional selector to increase specificity
+body .hds-side-nav__header-logo {
```

Comment:
this code is not anymore in Cloud UI, so we can remove this specificity "hacks"[^1] and move all the declarations under the same CSS selector (`.hds-side-nav__header-logo`)

[^1] this will be done in a different way, using this special Sass variable: https://github.com/hashicorp/structure/blob/main/packages/pds-ember/app/styles/pds/_overrides.scss#L8 which is used then in this way: https://github.com/hashicorp/cloud-ui/blob/main/packages/cloud-ui-core/addon/styles/_overrides.scss#L35)